pkg/garbagecollector/announcement_handlers.go

```diff
@@ -0,0 +1,67 @@
+package garbagecollector
```

Comment:
Since this is very specific to garbage collection on kubernetes, would you mind moving this under `pkg/kubernetes`.

Reply:
`pkg/garbagecollector/announcement_handlers.go` moved to `pkg/kubernetes`